powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/behavior/tasks/OperationServiceBehavior.java

```diff
@@ -1071,13 +1072,15 @@ private Map<String, Object> mapMerge(Map<String, Object> m1, Map<String, Object>
      */
     @SneakyThrows(GenericServiceException.class)
     private OperationDetailResponse convertFromEntityAndFillOtp(final OperationEntity source) {
-        final String totp = generateTotp(source, powerAuthServiceConfiguration.getProximityCheckOtpLength());
+        final int otpLength = powerAuthServiceConfiguration.getProximityCheckOtpLength();
+        final Duration otpStepLength = powerAuthServiceConfiguration.getProximityCheckStepLength();
+        final String totp = generateTotp(source, otpLength, otpStepLength);
```

Comment:
`otpLength` and `otpStepLength` - are confusing to me. But yes probably grammatically correct.